test/remote/ups_test.rb

```diff
@@ -393,6 +393,22 @@ def test_delivery_date_estimates_intl
     with_holidays(:ups) { assert_equal 1.business_days.after(today), ww_express_estimate.date }
   end
 
+  def test_delivery_date_estimates_to_have_valid_request_when_package_is_heavy_and_multiple_box
+    today = Date.current
+
+    response = @carrier.get_delivery_date_estimates(
+      location_fixtures[:new_york_with_name],
+      location_fixtures[:ottawa_with_name],
+      package_fixtures.values_at(:multiple_packages),
+      pickup_date=today,
+      {
+        :test => true
+      }
+    )
+    #this test is soly to test if the dates_request is valid, test would fail if missing or invalid request parameters.
```

Comment:
Comments explaining the purpose of a test are an indicator that the test purpose is unclear. If your test name is eloquent and descriptive, there should be no need for a comment.